pages/about.html

```diff
@@ -4,9 +4,53 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
 </head>
 <body>
-  
+  <header>
+    <nav class="nav-bar">
+      <a href="index.html"><i class="fa fa-home fa-lg"></i></a>
+      <ul class="nav-links">
+        <li class="nav-item"><a href="about.html">About Me</a></li>
+        <li class="nav-item"><a href="portfolio.html">Projects</a></li>
+      </ul>
+    </nav>
+  </header>
+  <div class="about-grid">
+    <section class="about-grid-left">
+    </section>
+    <section class="about-grid-name">
+      <p id="first-p">Diana</p>
+      <p>Martinez</p>
+    </section>
+    <section class="about-grid-side">
+      <div class="line"></div>
+      <h4>Welcome</h4>
```

Comment:
In semantic HTML, header tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)